src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -87,6 +87,11 @@ public static void main(String[] args) throws Exception {
             intervalMonitor = Integer.parseInt(argsLine.getOptionValue("im"));
         }
 
+        Boolean useVirtualThreads = false;
+        if (argsLine.hasOption("vt")) {
```

Comment:
Can you add a long form `--virtual-threads` option as well?

Please also make sure the `--help` output is updated.

Reply:
Ouch, I forgot to add the option at all.

```
usage: benchbase
...
 -vt,--virtual-threads <arg>    Use virtual threads instead of real
                                threads
```